EDEngineer/Resources/Data/entryData.json

```diff
@@ -1874,4 +1874,2476 @@
     ],
     "Group": "GuardianRuinsActive"
   }
-]
\ No newline at end of file
+]
+[
+    {
+    "Name": "Aerogel",
+    "ValueCr": 500,
+    "BarterCost": 9,
+    "BarterValue": 5,
+    "FormattedName": "aerogel",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Chemicals",
+    "SettlementType": [""],

```

Comment:
we should replace all the array containing empty string `[""]` with empty arrays `[]`

Reply:
Will do